index.js

```diff
@@ -1,9 +1,20 @@
-var crypto = require('crypto')
+var native = require('crypto')
+
+function nativePBKDF2 (password, salt, iterations, keylen, digest, callback) {
+  return native.pbkdf2(password, salt, iterations, keylen, digest || 'sha1', callback)
+}
+
+function nativePBKDF2Sync (password, salt, iterations, keylen, digest) {
+  return native.pbkdf2Sync(password, salt, iterations, keylen, digest || 'sha1')
+}
+
 /* istanbul ignore next */
-if (crypto && (!crypto.pbkdf2Sync || crypto.pbkdf2Sync.toString().indexOf('keylen, digest') === -1)) {
```

Comment:
`crypto` was removed here,  if it is false, then it would still explode below

Reply:
the issue is if it's undefined not if it's false

Reply:
`false/falsy/undefined`,  the point was the same,   it would explode with the same exception below

Reply:
`!native.pbkdf2Sync` will explode *here* if native is `undefined`,  so I guess I may be misunderstanding your fix

Reply:
The removal of `crypto &&` isn't a fix,  its removal of a pointless conditional.

Reply:
maybe the fix is for it to just give a useful error message if native is undefined instead of blowing up

Reply:
@calvinmetcalf out of interest,  is it a real error condition?
If so,  when?  Lets test for it :+1:

Reply:
if somebody was to require this directly, but prevent crypto from being loaded then crypto would be undefined here, now *why* I have no idea, probably in some weird enviroment

Reply:
@calvinmetcalf I don't think we need to support that... if it breaks,  we'll learn,  and patch...

Reply:
ok I'm still on -1 on this change that could potentially introduce new bugs for purely aesthetic reasons (as far as I can tell)

Reply:
OK,  I'll fix this up